panel/src/components/Forms/Writer/Toolbar.vue

```diff
@@ -68,145 +68,185 @@ export default {
 	},
 	computed: {
 		/**
-		 * The currently active node, if any
+		 * Currently active dropdown entry, if any
+		 * @returns {Object|undefined}
 		 */
-		activeNode() {
-			const nodes = Object.values(this.nodeButtons);
-			return nodes.find((button) => this.isNodeActive(button)) ?? false;
+		activeDropdownEntry() {
+			return Object.values(this.dropdownEntries).findLast(this.isNodeActive);
```

Comment:
`Explanation:` Using `findLast` here as the paragraph node will be also active in many block level nodes (e.g. list, quote) but we prefer to show the more specific one as active.